docs/visualize/lens.asciidoc

```diff
@@ -4,60 +4,51 @@
+a visualization that best dsiplays your data.
 
-* Quickly build visualizations by dragging and dropping data fields.
+With Lens, you can:
 
-* Understand your data with a summary view on each field.
+* Explore your data in just a few clicks.
 
-* Easily change the visualization type by selecting the automatically generated visualization suggestions.
+* Create visualizations with multiple layers and indices.
 
-* Save your visualization for use in a dashboard.
+* Use the automatically generated visualization suggestions to change the visualization type.
 
-[float]
-[[drag-drop]]
-=== Drag and drop
-
-The panel shows the data fields for the selected time period. When
-you drag a field from the data panel, Lens highlights where you can drop that field. The first time you drag a data field,
-you'll see two places highlighted in green:
-
-* The visualization builder pane
+* Add your visualizations to dashboards and Canvas workpads.
 
-* The *X-axis* or *Y-axis* fields
-
-You can incorporate many fields into your visualization, and Lens uses heuristics to decide how
-to apply each one to the visualization.
+To get started with *Lens*, click a field in the data panel, then drag and drop the field on a highlighted area.
 
 [role="screenshot"]
 image::images/lens_drag_drop.gif[]
 
-TIP: Drag-and-drop capabilities are available only when Lens knows how to use the data. You can still customize
-your visualization if Lens is unable to make a suggestion.
+You can incorporate many fields into your visualization, and Lens uses heuristics to decide how to apply each one to the visualization.
+
+TIP: Drag-and-drop capabilities are available only when Lens knows how to use the data. If *Lens* is unable to automatically generate a visualization,
+you can still configure the customization options for your visualization.
 
 [float]
 [[apply-lens-filters]]
-==== Find the right data
+==== Filter the data panel fields
 
-Lens shows you fields based on the <<index-patterns, index patterns>> you have defined in
-{kib}, and the current time range. When you change the index pattern or time filter,
-the list of fields are updated.
+The fields in the data panel based on your selected <<index-patterns, index pattern>>, and the <<set-time-filter,time filter>>. 
 
-To narrow the list of fields, you can:
+To change the index pattern, click it, then select a new one. The fields in the data panel automatically update.
 
-* Enter the field name in *Search field names*.
+To filter the fields in the data panel:
 
-* Click *Filter by type*, then select the filter. You can also select *Only show fields with data*
-to show the full list of fields from the index pattern.
+* Enter the name in *Search field names*.
+
+* Click *Filter by type*, then select the filter. To show all of the fields in the index pattern, select *Only show fields with data*.
 
 [float]
 [[view-data-summaries]]
-==== Data summaries
+==== View the top values
```

Comment:
It's not just showing top values, it also shows the distribution of numeric data. I don't think this section needed to be updated.